cmake/modules/FindTAEF.cmake

```diff
@@ -64,12 +64,13 @@ find_program(TAEF_EXECUTABLE te.exe PATHS
   ${TAEF_SDK_BIN}/${TAEF_BIN_ARCH}
   ${WINDOWS_KIT_10_PATH}
   ${WINDOWS_KIT_81_PATH}
+  $ENV{TAEF_PATH}
```

Comment:
If a user is explicitly setting an environment variable to find TAEF, shouldn't that be the first path we check not the last?

Reply:
Fixed